generators/express/index.ts

```diff
@@ -34,7 +34,7 @@ class ExpressGenerator extends PackageGenerator {
                 });
 
                 this.updateCircleCIConfig((config) => {
-                    config.workflows.build.jobs.deploy.requires.push(`${packageName}-archive`);
+                    config.workflows.build!.jobs.deploy!.requires.push(`${packageName}-archive`);
```

Comment:
In this case, we know that the keys are defined by the template that we juste rendered earlier.